static/app/utils/performance/quickTrace/traceFullQuery.tsx

```diff
@@ -96,12 +98,24 @@ function GenericTraceFullQuery<T>({
   );
 }
 
-export function TraceFullQuery(props: Omit<QueryProps<TraceFull[]>, 'detailed'>) {
-  return <GenericTraceFullQuery<TraceFull[]> {...props} detailed={false} />;
+export function TraceFullQuery(
+  props: Omit<QueryProps<TraceFull[] | TraceFullSplitResults>, 'detailed'>
+) {
+  return (
+    <GenericTraceFullQuery<TraceFull[] | TraceFullSplitResults>
+      {...props}
+      detailed={false}
+    />
+  );
 }
 
 export function TraceFullDetailedQuery(
-  props: Omit<QueryProps<TraceFullDetailed[]>, 'detailed'>
+  props: Omit<QueryProps<TraceFullDetailed[] | TraceDetailedSplitResults>, 'detailed'>
 ) {
-  return <GenericTraceFullQuery<TraceFullDetailed[]> {...props} detailed />;
+  return (
```

Comment:
We now respect that we can get two types of responses from the tracendpoint. For both detailed and non-detailed requests made to it.